src/pymanopt/optimizers/nelder_mead.py

```diff
@@ -12,14 +12,12 @@
 def compute_centroid(manifold, points):
     """Compute the centroid of `points` on the `manifold` as Karcher mean."""
 
-    @pymanopt.function.numpy(manifold)
     def objective(*y):
         if manifold.num_values == 1:
             (y,) = y
         return sum(manifold.dist(y, point) ** 2 for point in points) / 2
 
-    @pymanopt.function.numpy(manifold)
-    def gradient(*y):
+    def riemannian_gradient(*y):
```

Comment:
why did you change the name of the function?

Reply:
Simply because I saw below that the function was used as a `riemannian_gradient` when constructing the `Problem`. But maybe I misinterpreted something